components/Form/TextArea/TextArea.jsx

```diff
@@ -37,6 +37,7 @@ const TextArea = ({
       id={name}
       name={name}
       rows="5"
+      width="30"
```

Comment:
The `width` is not working, we can use `govuk-input govuk-input--width-${width}`.